src/keylessBackup/web3auth.ts

```diff
@@ -0,0 +1,57 @@
+import NodeDetailManager from '@toruslabs/fetch-node-details'
+import Torus from '@toruslabs/torus.js'
+import jwtDecode from 'jwt-decode'
+import Logger from 'src/utils/Logger'
+import { TORUS_NETWORK, TORUS_NETWORK_CONTRACT_ADDRESS, TORUS_SIGNER_BASE_URL } from 'src/config'
+
+const TAG = 'keylessBackup/torus'
+
+export async function getTorusPrivateKey({ verifier, jwt }: { verifier: string; jwt: string }) {
```

Comment:
this function is alarmingly complex, and relies on versions of deps that are already out of date, with breaking changes having taken place since then. however, getting it to work took *significant* effort, and I want to unblock the rest of this feature. I suggest we use this for now and I can try to upgrade it before we release to users (so it's still ok if the private keys returned by the refactored version are inconsistent with this method, if that happens for some reason). Ideally we'd use web3auth's nodeJS client, but it didn't work last time I tried it.

Reply:
I didn't unit test this for now because I'd rather just rewrite it than invest in mocks for behavior we don't really understand to begin with. But if anyone sees an opportunity for useful and meaningful unit test coverage I'm open to it.

Reply:
skipping unit test sounds good for now, especially if we are going to upgrade to a version with breaking changes. Maybe add a comment linking to a linear issue?

Reply:
done